test/unit/oatbox/user/UserLanguageServiceTest.php

```diff
@@ -70,15 +110,15 @@ public function testGetDataLanguage()
 
     public function testIsDataLanguageEnabled()
     {
-        $service = $this->getService();
+        $service = new UserLanguageService();
```

Comment:
Why we stoped using the previous method and now we are repeating the constructor several times?

Reply:
The previous method `getService()` just created and returned a new `UserLanguageService` instance with the provided options.

I found removing that indirection and instantiating the class unser test directly on each test case was a bit more clear (previously we called the `getService()` several times, now we directly call the constructor itself several times).

Reply:
I know, but if you have extra default parameters to the constructor os a default setter, etc you need to change in all places. 

I still find useful a factory method as it was before following DRY principle. It is a suggestion :)

Reply:
Okay, I'm restoring the previous method in 802e14877cb5ee2221a187526f4fb023eae5593c

Reply:
This was already restored ([here](https://github.com/oat-sa/generis/blob/feature/ADF-842-configurable-user-default-language/test/unit/oatbox/user/UserLanguageServiceTest.php#L149)), marking this thread as solved.